charts/stellar-network/values.yaml

```diff
@@ -1,25 +1,29 @@
 global:
   # namespace: default
-  networkPassphrase: "Test SDF Network ; September 2015"
-  horizonUrl: "https://horizon-testnet.stellar.org"
+  networkPassphrase: "Reynholm Industries Network ; January 2023"
```

Comment:
This should be overriden in the values-reynholmindustries.yaml file not in the horizon chart's values. We want to keep the helmcharts generic and elected to default to testnet for now.